runtime/src/lib.rs

```diff
@@ -1676,11 +1700,11 @@ mod tests {
 	#[test]
 	fn validate_transaction_submitter_bounds() {
 		fn is_submit_signed_transaction<T>()
-		where
-			T: CreateSignedTransaction<Call>,
+			where
+				T: CreateSignedTransaction<Call>,
```

Comment:
Is this change here by mistake? fix please.

Reply:
done